libs/routing/lanes/lane_way.hpp

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include "base/assert.hpp"
+
+#include <bitset>
+#include <initializer_list>
+#include <string>
+
+namespace routing::turns::lanes
+{
+enum class LaneWay : std::uint8_t
+{
+  None = 0,
+  Left,
+  SlightLeft,
+  SharpLeft,
+  Through,
+  Right,
+  SlightRight,
+  SharpRight,
+  Reverse,
```

Comment:
I know that you think of this in terms of logic, that reverse should have only single direction: either Left or Right and so we don't need to have both. But if my "lane arrow generation" code receives this LaneWay object, it would have no way of knowing whether this road is left-hand and right-hand. That's why I think it should contain both ReverseRight and ReverseLeft.